.github/workflows/pr-validation.yml

```
@@ -0,0 +1,157 @@
+# For more information see:
+# - https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+# - https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+
+name: pr-validation
+
+on:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.head_ref }}.${{ github.sha }}.pr-validation
+  cancel-in-progress: true
+
+jobs:
+  ## the reusable job will work after the PR is merged and files are committed to the master branch
+  # get-variables:
+  #   uses: ngxs/store/.github/workflows/get-variables.yml@master # <- references a reusable workflow file and a branch
```

Wouldn't it make more sense to reference the reusable workflow through an internal path for this head? Unless there is possibly a security risk in doing so?
I was thinking it would have been something like this:
```suggestion
  #   uses: ./.github/workflows/get-variables.yml
```